lib/wanda_web/plugs/api_redirector.ex

```diff
@@ -45,7 +45,7 @@ defmodule WandaWeb.Plugs.ApiRedirector do
       nil ->
         conn
         |> put_resp_content_type("application/json")
-        |> resp(:not_found, Jason.encode!(ErrorView.render("404.json", %{detail: "Not found"})))
+        |> resp(:not_found, Jason.encode!(ErrorJSON.render("404.json", %{detail: "Not found"})))
```

Comment:
Most probably something like this could more appropriate:
```
...
|> put_resp_content_type("application/json")
|> put_status(:not_found)
|> put_view(json: ErrorJSON)
|> render(:"404")
|> halt()
|> render(

```

What do you think? To use the same convention we use in everywhere else

Reply:
We don't use it in web in a similar function but i see what you mean, it just throws error, going to have a look.
For me both is good :)